src/components/common/ButtonComponent.jsx

```diff
@@ -10,8 +10,8 @@ import styled from 'styled-components';
 //
 // 예시 <Button btnInfo={{ text: '다음', color: 'jade' }} />
 
-const Button = ({ btnInfo }) => {
-  const { text, color, width, onClick } = btnInfo;
+const Button = ({ btnInfo, onClick, type }) => {
```

Comment:
제가 착각했습니다 이렇게 전달하면 명시하지 않은 props 모두 한 번에 전달된다고 해요!

```suggestion
const Button = ({ btnInfo, ...props }) => {
```